src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -3202,6 +3202,24 @@ public class Counter : ComponentBase
             AssertCSharpDocumentMatchesBaseline(generated.CodeDocument);
             CompileToAssembly(generated);
         }
+
+        [Fact]
+        public void ComponentImports()
+        {
+            // Arrange & Act
+            var generated = CompileToCSharp("_Imports.razor", @"
+@using System.Text
+@using System.Reflection
+
+@Foo
+<div>Hello</div>
+", throwOnFailure: false, fileKind: FileKinds.ComponentImport);
+
+            // Assert
+            AssertDocumentNodeMatchesBaseline(generated.CodeDocument);
+            AssertCSharpDocumentMatchesBaseline(generated.CodeDocument);
+            CompileToAssembly(generated, throwOnFailure: false);
+        }
```

Comment:
Should add a test (or update this one) to use some component-specific directives.

Reply:
still don't see one of these.

Reply:
nvm its down below